pkger/models.go

```diff
@@ -750,6 +753,8 @@ type chart struct {
 	XCol, YCol    string
 	XPos, YPos    int
 	Height, Width int
+
+	BinSize int32
```

Comment:
make this type `int` and convert it on https://github.com/influxdata/influxdb/pull/15857/files#diff-26b9adadf7ef8d11b67d2d091e56b593R799

```go
BinSize:           int32(c.BinSize)
```

I want to keep the integration types as vanilla as possible right now.